src/client/errorMiddleware.js

```diff
@@ -1,7 +1,7 @@
 import { notify } from './app/Notification/notificationActions';
 
 function parseBlockChainError(error) {
-  return error.split(':').slice(-1)[0];
+  return error.split(':')[2].split('\n')[0].substring(1);
```

Comment:
This is assuming error is always going to be a string, but hopefully it always is, I would suggest using lodash since it already comes with polyfills and errorchecks and maybe splitting this up into a few different lines that way they aren't reliant too much on each other. Since right now for the first `.split` to work then `error` needs to be a string. Then if that passes but the array results for the index `2` is undefined then it will throw an error because it expects the array result for the index `2` to be a string so you can split it again..

So I think you can do something like this 
```
const errorArray = _.split(":");
const fullErrorText = _.nth(errorArray, 2);
let errorMessage = "  " 
if (!_.isEmpty(fullErrorText)) {
  errorMessage = _.split(fullErrorText, '\n')
  errorTest = _.nth(errorMessage, 0) } || " ";
  return  errorTest.substring(1); // or maybe lodash has something
}
```

Or something like, idk though, your call.. I just quickly did that example so I could be wrong on that lol